build/entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+sudo update-ca-certificates --fresh >/dev/null
```

Comment:
This should be in `Dockerfile` rather than entrypoint.

Reply:
Thanks for the code review and comments. @khos2ow 
I think it's important to run it in an entrypoint instead of a Dockerfile.

If team A and team B require different organizational root certificates and self-certificates, running Dockerfile will require multiple images with each certificate statically embedded, and many image checks will be required.

If you run update-ca-certificates dynamically at container execution time with entrypoint, you only need one container image.

I think it is better to use certificates that expire and revoke dynamically and flexibly, instead of statically including them in the container image.
This way, there is no need to rebuild the container image every time a certificate expires and revokes.

Do you have any other good ideas to solve this problem?

Reply:
Sorry for the massive delay! Your suggestions sounds good to me.

Reply:
@cpanato wdyt?

Reply:
How I think it should be used.
Example of docker-compose.yml for use with a single container image
```yaml
services:
  db:
    image: postgres:latest
<snip>
 mattermost:
    image: mattermost/mattermost-team-edition:same-version
    volumes:
      - ./certs/team-A.crt:/usr/local/share/ca-certificates/team-A.crt ### <- root certificates for team-A
      #- ./certs/team-B.crt:/usr/local/share/ca-certificates/team-B.crt ### <- root certificates for team-B
      - ./volumes/app/mattermost/config:/mattermost/config:rw
      - ./volumes/app/mattermost/data:/mattermost/data:rw
      - ./volumes/app/mattermost/logs:/mattermost/logs:rw
      - ./volumes/app/mattermost/plugins:/mattermost/plugins:rw
      - ./volumes/app/mattermost/client-plugins:/mattermost/client/plugins:rw
      - /etc/localtime:/etc/localtime:ro
    environment:
      - MM_USERNAME=mmuser
      - MM_PASSWORD=mmuser_password
      - MM_DBNAME=mattermost
```

Reply:
I'm not too fond of the idea to have the mattermost user a sudo permission to execute any command as root. This can lead to some malicious attack if the attacker has access to the container.

If need to add custom certificates, you can have a pipeline to rebuild the docker image and adding your custom CA there. I see this as common practices for other images

any thoughts @jwilander @stylianosrigas

Reply:
+1 that we should not have mattermost user a sudoer